contrib/src/main/java/com/datatorrent/contrib/kafka/AbstractKafkaInputOperator.java

```diff
@@ -188,6 +188,8 @@
   @Min(1)
   private int initialPartitionCount = 1;
 
+  private boolean isPartitionBasedOnLoad = false;
```

Comment:
Is this can be achieved as follows. Please correct it, if I am wrong.
if (msgRateUpperBound != Long.MAX_VALUE) {
/// Dynamic partition based on load is enabled
} else {
/// Dynamic partition based on load is disabled
}